util/builtin_operators.go

```diff
@@ -251,6 +252,28 @@ func IPMatchFunc(args ...interface{}) (interface{}, error) {
 	return bool(IPMatch(ip1, ip2)), nil
 }
 
+func GlobMatch(key1 string, key2 string) bool {
+	ok, err := path.Match(key2, key1)
+	if err != nil {
+		panic(err)
```

Comment:
We should avoid the `panic`.

Reply:
Other functions like `IPMatch()` also handle errors like this. 

I just want this function to meet the matcherFunc in the defaultRoleManager. I can't think of a better way to handle errors without changing this function type. But this will cause all users to override their custom matchfunc

Reply:
I suggest returns the `(bool, error)`.

Reply:
Ok,  I have modified

Reply:
@dovics Only changes the `GlobMatch`.